apps/ledger-live-mobile/src/components/Delta.tsx

```diff
@@ -19,6 +19,7 @@ type Props = {
   /** whether to show a placeholder in case the percent value is not valid */
   fallbackToPercentPlaceholder?: boolean;
   textProperties?: Partial<BaseTextProps>;
+  isLargeMover?: boolean;
```

Comment:
I would rename this parameter to something more generic as it may be used in other features. Maybe something like `isPercentSignDisplayed`, wdyt ?